pallets/omnipool/src/lib.rs

```diff
@@ -1659,4 +1673,111 @@ impl<T: Config> Pallet<T> {
 
 		Err(Error::<T>::NotAllowed.into())
 	}
+
+	/// Remove asset from list of Omnipool assets.
+	pub fn remove_asset(asset_id: T::AssetId) -> DispatchResult {
```

Comment:
We can emit new `TokenRemoved` event from this one. Other methods can emit events as well.

Reply:
this could be confusing, because current use case does not really remove the asset, it is just moved to subpool. 

subpool implemenation actually emit an event "AssetMigrated"

Reply:
It's a public interface. Add there a comment that these methods modify the storage and don't emit any event, so any consumer is aware of it.